roles/ceph-rgw/tasks/rgw_pool_pgs.yml

```diff
@@ -2,7 +2,7 @@
 - name: create rgw pools if create_pools is defined
   command: ceph --connect-timeout 5 --cluster {{ cluster }} osd pool create {{ item.key }} {{ item.value.pg_num }}
   changed_when: false
-  failed_when: false
+  run_once: true
   with_dict: "{{ create_pools }}"
```

Comment:
insert a `delegate_to: "{{ groups[mon_group_name][0] }}"`

Reply:
@guits Just curious. Whats the point of having "mon_group_name", and the rest of the group names variables,  if we are forced to hardcode the default group names (mons, rgws, etc) in the main playbook.

https://github.com/ceph/ceph-ansible/blob/master/site-docker.yml.sample#L4

Reply:
Btw, per this PR https://github.com/ceph/ceph-ansible/pull/2428/files, which copies client.admin.keyring to RGWs, I think there is no need to delegate the task to monitors anymore as RGWs could execute it. Am I wrong?

Reply:
@jtudelag it let users the possibility to rename their groups as they want.
The 2 main playbooks in the repository are just provided as sample where you would have to rename the group in any case if you decide to use them with custom group names except if you keep the default values.

For instance, if you want to rename your monitor group you will have only few occurrences to edit whereas you will have more than 100 in the whole playbook as you can see below :

```
 guits@work:(master)~/GIT/ceph-ansible$ git grep 'mons' site*.sample | wc -l 
4
 guits@work:(master)~/GIT/ceph-ansible$ git grep 'mon_group_name' | wc -l
148
```

Reply:
@jtudelag there is a condition to copy the admin keyring so if you don't set `copy_admin_key: true` you won't have it on your rgw node.

Reply:
@guits Thanks for the explanations! So I guess we can keep it this way. Running it delegated to the first mon guarantees the admin keyring is present and the command can be executed.